dev-docs/bidders/mantis.md

```diff
@@ -1,23 +1,19 @@
 ---
 layout: bidder
-title: Mantis
+title: MANTIS Ad Network
 description: Prebid Mantis Bidder Adaptor
-
 top_nav_section: dev_docs
 nav_section: reference
-
 hide: true
-
 biddercode: mantis
-
 biddercode_longer_than_12: false
 
```

Comment:
`prebid_1_0_supported : true` should be added now with https://github.com/prebid/Prebid.js/pull/1840 merged

Reply:
+1  without that key in the YAML header this adapter won't appear in the list of 1.0+ adapters on http://prebid.org/download.html

Reply:
updated, thanks